apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/MoreActions/FinalDecisionDialog.tsx

```diff
@@ -0,0 +1,339 @@
+                  description: "Final assessment decision input",
+                  id: "GGPWwr",
+                })}
+                rules={{
+                  required: intl.formatMessage(errorMessages.required),
+                }}
+                items={[
+                  {
+                    value: "qualified",
+                    label: intl.formatMessage({
+                      defaultMessage: "Qualify candidate",
+                      description: "Qualify candidate option",
+                      id: "EvR6qK",
+                    }),
+                  },
+                  {
+                    value: "disqualified",
+                    label: intl.formatMessage({
+                      defaultMessage: "Disqualify candidate",
+                      description: "Disqualify candidate option",
+                      id: "oIM22z",
+                    }),
+                  },
+                ]}
+              />
+              {finalAssessmentDecisionValue === "disqualified" && (
+                <RadioGroup
+                  idPrefix="disqualifiedDecision"
+                  name="disqualifiedDecision"
+                  legend={intl.formatMessage({
+                    defaultMessage: "Disqualified decision",
+                    description: "Disqualified decision input",
+                    id: "Ed+xy1",
+                  })}
+                  rules={{
+                    required: intl.formatMessage(errorMessages.required),
+                  }}
+                  items={[
+                    {
+                      value: "application",
+                      label: intl.formatMessage({
+                        defaultMessage: "Screened out on application",
+                        description: "Screened out on application option",
+                        id: "YueN6y",
+                      }),
+                    },
+                    {
+                      value: "assessment",
+                      label: intl.formatMessage({
+                        defaultMessage: "Unsuccessful during assessment",
+                        description: "Unsuccessful during assessment option",
+                        id: "wrqRvV",
+                      }),
+                    },
+                  ]}
+                  data-h2-margin-top="base(x1)"
+                />
+              )}
+              {finalAssessmentDecisionValue === "qualified" && (
+                <>
+                  <Heading
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin-bottom="base(x.5)"
+                    data-h2-font-size="base(normal)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "Select an expiry date",
+                      id: "3MboNR",
+                      description:
+                        "Label for date selection input for expiry date",
+                    })}
+                  </Heading>
```

Comment:
I'd like to avoid this heading all together and made a suggesiton on how but, if we do need it it should have level="h4" size="h6".

Also, now that I mention it... We would have two levels of headings at the same size in this dialog so we may want to consult designers to see if we can increase the other ones to `h5`. I see this one is supposed to be body but I don't believe we can do that with headings.